### PR TITLE
Add camelCase notation to code style

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | -

### CODE REVIEW

File: 010-code-style.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Code formatting used in Yii 3 packages is based on [PSR-1](https://www.php-fig.o
 ## Names
 
 - Use English only.
+- Use camelCase notation, including abbreviations.
```
Comment (arogachev) [inline 010-code-style.md:9]: Maybe provide an example such as `enableIdn`?

Comment (vjik) [reply 010-code-style.md]: done

**Review by arogachev — APPROVED** [review]

File: 010-code-style.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Code formatting used in Yii 3 packages is based on [PSR-1](https://www.php-fig.o
 ## Names
 
 - Use English only.
+- Use camelCase notation, including abbreviations (e. g., `enableIdn`).
```
Comment (samdark) [inline 010-code-style.md:9]:
```suggestion
- Use camelCase notation, including abbreviations (e.g., `enableIdn`).
```

**Comment by samdark** [comment]
👍

**Pull request merged**